src/views/user/utils/FormField.vue

```diff
@@ -40,6 +40,14 @@
         class="input"
         :class="{ 'is-danger': hasError || errorMessage }"
       />
+      <span
+        v-if="type === 'password'"
+        class="icon is-small is-right"
+        @click="onMaskToggle"
+        style="pointer-events: initial"
+      >
+        <fa-icon :icon="unmasked ? 'eye-slash' : 'eye'" />
```

Comment:
One could add a title to the input so that the action is explicit : `:title="$gettext(unmasked ? 'Hide password' : 'Show Password', 'Passord prompt')"`